codegenerator/cli/templates/dynamic/codegen/src/Types.res.hbs

```diff
@@ -549,6 +552,16 @@ type fuelContractConfig = {
   events: array<fuelEventConfig>,
 }
 
+type fuelSupplyParams = {
+  subId: string,
+  amount: bigint,
+}
+
+let fuelSupplyParamsSchema = S.object(s => {
+  subId: s.field("subId", S.string),
+  amount: s.field("amount", BigInt.schema),
+})
```

Comment:
Out of interest why not use S.schema for this? Is it slower than S.object?

Reply:
It's slower on schema creating, when parsing it's the same. I think `S.schema` would be fine here. I just used `S.object` out of habit 😅